static/app/views/settings/projectMetrics/metricsExtractionRuleForm.tsx

```diff
@@ -133,20 +142,16 @@ export function MetricsExtractionRuleForm({isEdit, project, onSubmit, ...props}:
             name="type"
             disabled={isEdit}
             options={TYPE_OPTIONS}
-            label={t('Type')}
-            help={t(
-              'The type of the metric determines which aggregation functions are available and what types of values it can store. For more information, read our docs'
-            )}
+            label={t('Aggregate')}
+            help={t('Select the aggregations you want to store.')}
```

Comment:
That is what I meant in the stand-up. It does not make sense yet to name it aggregate as the user still chooses the type of the metric in this version.
No hard opinion here as it is still feature flagged and WIP 🤷

Reply:
@ale-cota ^

Reply:
@priscilawebdev just checked with Arthur and since his PR might take longer to merge (waiting for back-end), let's leave the copy here as is and he can update it there.